Packages.props

```diff
@@ -4,7 +4,7 @@
 		<PackageReference Update="Autofac" Version="$(AutofacVersion)" />
 		<PackageReference Update="Autofac" Version="[5.1.2, 7)" Condition="'$(AutofacVersion)' == '' Or '$(AutofacVersion)' == 'lowestSupported'" />
 		<PackageReference Update="Microsoft.NET.Test.Sdk" Version="[15.9.2, 16)" />
-		<PackageReference Update="System.Collections.Immutable" Version="[1.6, 2)" />
+		<PackageReference Update="System.Collections.Immutable" Version="[1.6, 6)" />
```

Comment:
I would fix this problem once and for all and remove the upper bound. Packages from Microsoft like this usually don't have breaking changes, right?